tests/cases/conformance/jsdoc/checkJsdocSatisfiesTag12.ts

```diff
@@ -0,0 +1,54 @@
+// @noEmit: true
+// @allowJS: true
+// @checkJs: true
+
+// @filename: /a.js
+/**
+ * @typedef {Object} T1
+ * @property {number} a
+ */
+
+/**
+ * @typedef {Object} T2
+ * @property {string} a
+ */
+
+/**
+ * @typedef {Object} T3
```

Comment:
I noticed that `T3` wasn't used when it was added to the last commit. Did you mean to test it out on `t7`?

```ts
/**
 * @satisfies {T2}
 */
const t7 = { a: "a" };
```

Reply:
You are right. Fixed this typo, thanks!